lib/koans/05_lists.ex

```diff
@@ -9,10 +9,6 @@ defmodule Lists do
     assert List.last([1, 2, 3]) == :__
   end
 
-  koan "Diversity is embraced" do
```

Comment:
Ah, I didn't realise that this is also the only koan with `++`. Will put it back in